.ci/install_kata_kernel.sh

```diff
@@ -98,16 +101,15 @@ main() {
 	download_repo
 	kernel_version="$(get_current_kernel_version)"
 	kata_config_version="$(get_kata_config_version)"
-	current_kernel_version="${kernel_version}.${kata_config_version}"
-	info "Current Kernel version ${current_kernel_version}"
-	info "Get packaged kernel version"
-	packaged_kernel_version=$(get_packaged_kernel_version)
-	info "Packaged Kernel version ${packaged_kernel_version}"
-	if [ "$packaged_kernel_version" == "$current_kernel_version" ] && [ "$kernel_arch" == "x86_64" ]; then
-		# If installing packaged kernel from OBS fails,
+	current_kernel_version="${kernel_version}-${kata_config_version}"
+	last_build_kernel_version=$(curl -sfL "${latest_build_url}/latest")
```

Comment:
I think you do this same line again later on (line 72 in current patchset)? - this does not look like a local var - can you just re-share this value, to save an extra duplicate `curl`?